pkg/driver/deployment_awslambda.go

```diff
@@ -48,10 +48,7 @@ func DeployFunctionsAWSLambda(functions []*common.Function) {
 					} else {
 						atomic.AddUint64(&counter, 1)
 						for i := 0; i < len(functionGroup); i++ {
-							// Extract 0 from trace-func-0-2642643831809466437 by splitting on "-"
-							shortName := strings.Split(functionGroup[i].Name, "-")[2]
-
-							functionGroup[i].Endpoint = functionToURLMapping[shortName]
+							functionGroup[i].Endpoint = functionToURLMapping[strconv.Itoa(i)]
```

Comment:
How would it work with several batches of functions to deploy? Wouldn't it have the same iterator for different batches?

Reply:
As discussed, will make the edit to minimise confusion for code clarity. Thanks!